third_party/VENDOR-LICENSE/github.com/hashicorp/golang-lru/arc.go

```diff
@@ -0,0 +1,257 @@
+package lru
```

Comment:
this file looks bogus

Reply:
As it turns out, according to https://github.com/google/go-licenses/pull/28 its because that for "Reciprocal" licenses the whole source code is copied (regardless whether it has been changed or not).